lib/rails_admin/adapters/mongoid.rb

```diff
@@ -19,9 +22,12 @@ def new(params = {})
       end
 
       def get(id)
-        AbstractObject.new(model.find(id))
+        model_instance = model.find(id)
+        raise RaiseNotErrorFalse if model_instance.nil?
```

Comment:
Introducing new error class is unnecessary, just returning here should be enough.